rules/opentitan.bzl

```diff
@@ -446,9 +519,9 @@ def opentitan_rom_binary(
     This macro is mostly a wrapper around a opentitan_binary macro, which itself
     is a wrapper around cc_binary, but also creates artifacts for each of the
     keys in `per_device_deps`. The actual artifacts created are an ELF file, a
-    BIN file, the disassembly, and the scrambled (ROM) VMEM file. Each of these
-    output targets performs a bazel transition to the RV32I toolchain to build
-    the target under the correct compiler.
+    BIN file, the disassembly, the sim_dv logs database, and the scrambled (ROM)
+    VMEM file. Each of these output targets performs a bazel transition to the
+    RV32I toolchain to build the target under the correct compiler.
```

Comment:
The bit about the "transition to RV32I toolchain" was already added. I didn't add it. It just looks like I did in the diff because I moved some new line chars around. Happy to change this text thought if you prefer.

Reply:
Nope, feel free to ignore every Nit of mine. If I feel strong about something being fixed I wont mark it that way.